components/selection/selection-select-all.js

```diff
@@ -44,7 +44,7 @@ class SelectAll extends FocusMixin(LocalizeCoreElement(SelectionObserverMixin(Li
 	}
 
 	render() {
-		if (this._provider && this._provider.selectionSingle) return;
+		if (!this._provider || this._provider.selectionSingle) return;
```

Comment:
Dave B suggested leaving this unchanged (still render when there's no selection provider), but I investigated a bit further and found:

- we definitely don't want it to render in the selection-controls
- [`selection-input` doesn't render with no provider](https://github.com/BrightspaceUI/core/blob/main/components/selection/selection-input.js#L93)

So it seems reasonable to not render this when there's no selection provider detected.